src/main/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedProcessor.java

```diff
@@ -58,7 +61,16 @@ public void processQuestionnaireLinked(ResponseManagementEvent questionnaireLink
     UacQidLink uacQidLink = uacQidLinkOpt.get();
     Case caze = caseOpt.get();
 
+    // If already receipted
+    if (!uacQidLink.isActive()) {
```

Comment:
Check `&& !caze.ReceiptReceived()` to ensure that we don't publish a duplicate case updated event

Reply:
done

Reply:
sorry, try now